bin/p2-bootstrap/bootstrap.go

```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 	"time"
 
-	"github.com/square/p2/pkg/intent"
+	"github.com/square/p2/pkg/p2consul"
```

Comment:
wait, `store` would probably collide with the local variable `store := p2consul.NewStore(...)`. Maybe something like `kv`?